README.md

```diff
@@ -38,332 +40,374 @@ You need to install these cards first before using this strategy
-| `icon_color`          | string          | Optional    | Icon color to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                        |
-| `primary`             | string          | Optional    | Primary info to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                      |
-| `secondary`           | string          | Optional    | Secondary info to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                    |
-| `badge_icon`          | string          | Optional    | Badge icon to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                        |
-| `badge_color`         | string          | Optional    | Badge icon color to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                  |
-| `picture`             | string          | Optional    | Picture to render. May contain [templates](https://www.home-assistant.io/docs/configuration/templating/).                           |
-| `multiline_secondary` | boolean         | `false`     | Enables support for multiline text for the secondary info.                                                                          |
-| `layout`              | string          | Optional    | Layout of the card. Vertical, horizontal and default layout are supported                                                           |
-| `fill_container`      | boolean         | `false`     | Fill container or not. Useful when card is in a grid, vertical or horizontal layout                                                 |
-| `tap_action`          | action          | `none`      | Home assistant action to perform on tap                                                                                             |
-| `hold_action`         | action          | `none`      | Home assistant action to perform on hold                                                                                            |
-| `entity_id`           | `string` `list` | Optional    | Only reacts to the state changes of these entities. This can be used if the automatic analysis fails to find all relevant entities. |
-| `double_tap_action`   | action          | `more-info` | Home assistant action to perform on double_tap                                                                                      |
-| `extra_cards`         | list of cards   | Optional    | A list of cards to show on the top of the area subview                                                                              |
+The area object includes all options from the template mushroom card and `extra_cards` which is a list of cards to show
+at the top of the area subview.
+The order of definition is used to sort the rooms and pre-built views
+
+[//]: # (TODO: Describe the default values of below table.)
```

Comment:
@AalianKhan Please follow up line 138.

Reply:
It could be. I'm asking you because I haven't made any use of custom configurations yet.
Previously, the default value was described as optional.
I've changed this, by best educated guess, to an actual value instead of telling if it's optional or not.

However, I'm not truly sure if 'unset or empty' is correct.
My proposal is based on allowing the property not to be set at all or setting it to an empty value.
E.g. `icon: ""` or `entity_id: []`
If an unset or empty value leads to unexpected behavior, I've proposed something wrong and the description has to be corrected.

Reply:
If you think the current changes are OK after all, you can click the "Resolve conversation"button below. :)